extension/data/modules/modmatrix.js

```diff
@@ -735,7 +735,7 @@ function modmatrix () {
         const nerActive = $nerNext.length;
         let loadComments = false;
         const parser = SnuOwnd.getParser(SnuOwnd.getRedditRenderer());
-        $('.content .menuarea').append('<div class="spacer"><a href="javascript:;" class="activate-comment-load tb-general-button" >Load text of removed comments.</a></div>');
+        $('.content .menuarea').append('<div class="spacer"><a href="javascript:;" class="activate-comment-load tb-general-button" >Load text of comments.</a></div>');
```

Comment:
While we're here, button text shouldn't have periods.

```suggestion
        $('.content .menuarea').append('<div class="spacer"><a href="javascript:;" class="activate-comment-load tb-general-button" >Load text of comments</a></div>');
```